python/lsst/afw/table/baseColumnView.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+
+import numpy as np
+
+from ._baseColumnView import *
+
+def _get(self, key):
```

Comment:
There is no way to directly extend a Python class in Python either. And this is the standard way to do it, although it can be hidden behind a class decorator. In fact Nate and I recently wrote a PEP specifically to fix this issue (which was unfortunately rejected by the Python developers). In the current Swig code this approach is used as well, although Swig itself has `%extends` with `%pythoncode` which is used in most places.